nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -26,7 +26,7 @@
     <file src="Microsoft.Windows.CsWinRT.Prerelease*.targets" target="build"/>
     <file src="Microsoft.Windows.CsWinRT.Authoring.Transitive.targets" target="build"/>
     <file src="Microsoft.Windows.CsWinRT.Authoring.Transitive.targets" target="buildTransitive"/>
-    <file src="$cswinrt_exe$\..\WinRT.Interop.winmd" target="metadata"/>
+    <file src="$cswinrt_exe$\..\..\obj\merged\WinRT.Interop.winmd" target="metadata"/>
```

Comment:
should we instead be picking it up for something like bin rather than obj?

Reply:
that's how it was to begin with - it's really the same file